src/components/credentials/credentialListItem.js

```diff
@@ -127,21 +128,15 @@ class CredentialListItem extends React.Component {
       case 'sources':
         (item.sources || []).sort((item1, item2) => item1.name.localeCompare(item2.name));
         return (
-          <Grid fluid>
-            {item.sources &&
-              item.sources.map(source => (
-                <Grid.Row key={source.name}>
-                  <Grid.Col xs={12} sm={4}>
-                    <span>
-                      <Tooltip tooltip={dictionary[source.source_type]}>
-                        <Icon type={typeIcon.type} name={typeIcon.name} />
-                      </Tooltip>
-                      &nbsp; {source.name}
-                    </span>
-                  </Grid.Col>
-                </Grid.Row>
-              ))}
-          </Grid>
+          <List isPlain>
+            {item?.sources?.map(source => (
+              <Tooltip tooltip={dictionary[source.source_type]}>
+                <ListItem key={source.name} icon={<Icon type={typeIcon.type} name={typeIcon.name} />}>
+                  {source.name}
+                </ListItem>
+              </Tooltip>
+            ))}
+          </List>
```

Comment:
good effort on trying to get the tooltip in there! 

- unfortunately it jams a `span` around the `li` tag, 
- and it centers the tooltip on the element... which in this case has its width set by the largest list item so we'd also need to set the tooltip alignment to the left... 
 
...so lets avoid all of that

required:
- flip this back over to just wrapping the copy... 
   - not as cool, but maybe at some point if the tooltip gets upgraded to PF4 it won't wrap the `span` tag on there

```
        return (
          <List isPlain>
            {item?.sources?.map(source => (
              <ListItem key={source.name} icon={<Icon type={typeIcon.type} name={typeIcon.name} />}>
                <Tooltip tooltip={dictionary[source.source_type]}>{source.name}</Tooltip>
              </ListItem>
            ))}
          </List>
        );

Reply:
Good point - updated!